engine-standalone-storage/src/sync/mod.rs

```diff
@@ -234,6 +237,7 @@ fn non_submit_execute<'db>(
             None
         }
 
+        TransactionKind::FtTransferCall(_) if is_disabled_legacy_nep141 => None,
```

Comment:
Please also add this for the other NEP-141 methods that become no-op after the split.